data-access/persoon.ts

```diff
@@ -204,3 +225,157 @@ export async function copyPerson(subject: Term, graph: Term) {
     throw Error(`Could not copy person with uri: ${escaped.person}`);
   }
 }
+
+async function exists(personId: string): Promise<boolean> {
+  const askIfExists = `
+      PREFIX person: <http://www.w3.org/ns/person#>
+      PREFIX mu: <http://mu.semte.ch/vocabularies/core/>
+
+      ASK {
+        GRAPH ?personGraph {
+          ?person a person:Person;
+            mu:uuid ${sparqlEscapeString(personId)}.
+        }
+
+        FILTER ( ?personGraph != <http://mu.semte.ch/vocabularies/ext/FormHistory>)
+      }
+    `;
+
+  const result = await querySudo(askIfExists);
+
+  return getBooleanSparqlResult(result);
+}
+
+async function findOnafhankelijkeFractieUri(
```

Comment:
I think this gives issues with bestuursperiodes. If you have an onafhankelijkeFractie in a different bestuursperiode, this will also return it. 
Either we need to restrict this to the current bestuursperiode or we need to give an extra argument (bestuursperiode)